aces/analysis/giantcube_cuts.py

```diff
@@ -292,10 +292,12 @@ def do_all_stats(cube, molname, mompath=f'{basepath}/mosaics/cubes/moments/',
 
     print(f"Howargs: {howargs}.  Using vmask.")
 
-    if molname in ('H13CO+', 'HC15N'):
+    if molname in ('HC15N'):
         vmask = velocity_mask(cube, vmin_pos=0 * u.km/u.s, vmin_neg=-100 * u.km/u.s)
+    elif molname in ('H13CO+', ):
+        vmask = velocity_mask(cube, vmin_pos=-50 * u.km/u.s, vmin_neg=-150 * u.km/u.s)
     elif molname in ('SO21', ):
-        vmask = velocity_mask(cube, vmax_pos=100 * u.km/u.s, vmax_neg=0 * u.km/u.s)
+        vmask = velocity_mask(cube, vmax_pos=150 * u.km/u.s, vmax_neg=50 * u.km/u.s)
```

Comment:
you sure it should only go to +50?  I can do that, it just seems like it'll miss almost everything